### PR TITLE
sysbuild: Add missing board configurations

### PR DESCRIPTION
**Pull request by nordicjm**

### CODE REVIEW

**Review by MarkusLassila — APPROVED** [review]
SLM is OK

File: boards/nordic/thingy91x/thingy91x_nrf5340_cpuapp_common.dts [hunk 1]
```diff
@@ -136,6 +136,10 @@
 zephyr_udc0: &usbd {
 	compatible = "nordic,nrf-usbd";
 	status = "okay";
+
+	cdc_acm_uart: cdc_acm_uart {
```
Comment (maxd-nordic) [inline boards/nordic/thingy91x/thingy91x_nrf5340_cpuapp_common.dts:140]: To match this with the connectivity_bridge setup:
```suggestion
	cdc_acm_uart0: cdc_acm_uart0 {
```

File: applications/asset_tracker_v2/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# MCUBoot settings
+CONFIG_BOOT_MAX_IMG_SECTORS=512
+
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_SPI_NOR_SFDP_DEVICETREE=y
+CONFIG_MULTITHREADING=y
```
Comment (maxd-nordic) [inline applications/asset_tracker_v2/sysbuild/mcuboot/boards/thingy91x_nrf9151.conf:8]:
```suggestion
# MCUBoot settings
CONFIG_BOOT_MAX_IMG_SECTORS=512

CONFIG_SPI=y
CONFIG_SPI_NOR=y
CONFIG_SPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
CONFIG_SPI_NOR_SFDP_DEVICETREE=y
CONFIG_MULTITHREADING=y

# Disable Zephyr console and use UART for MCUboot serial recovery instead
CONFIG_CONSOLE=n
CONFIG_CONSOLE_HANDLER=n
CONFIG_UART_CONSOLE=n
CONFIG_MCUBOOT_SERIAL=y

CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD=y
CONFIG_BOOT_SERIAL_IMG_GRP_IMAGE_STATE=y
```

**Review by plskeggs — APPROVED** [review]

**Comment by MarkusLassila** [comment]
@maxd-nordic: Some thingy91x targets (such as samples/wifi/scan/ sample.thingy91x_nrf7002.scan) are failing to build with following error:

/home/ncs/toolchains/674f7b971c/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/zephyr_pre0.elf section `rodata' will not fit in region `FLASH'
/home/ncs/toolchains/674f7b971c/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: region `FLASH' overflowed by 960 bytes

**Comment by maxd-nordic** [comment]
> @maxd-nordic: Some thingy91x targets (such as samples/wifi/scan/ sample.thingy91x_nrf7002.scan) are failing to build with following error:
> 
> /home/ncs/toolchains/674f7b971c/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: zephyr/zephyr_pre0.elf section `rodata' will not fit in region `FLASH' /home/ncs/toolchains/674f7b971c/opt/zephyr-sdk/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: region `FLASH' overflowed by 960 bytes

This worked fine with about 30% flash usage on b74fcc2731952a45ff92e5ff37ddbce01e88b2d0.
Now, it also fails with a different error on main.
Would be nice if at least the build-only tests would be checked before forcing changes into NCS...

**Review by maxd-nordic — APPROVED** [review]

**Pull request merged**